api/src/Entity/WebsiteMedia.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Entity;
+
+use ApiPlatform\Doctrine\Orm\Filter\OrderFilter;
+use ApiPlatform\Doctrine\Orm\Filter\SearchFilter;
+use ApiPlatform\Metadata\ApiFilter;
+use ApiPlatform\Metadata\ApiProperty;
+use ApiPlatform\Metadata\ApiResource;
+use ApiPlatform\Metadata\Delete;
+use ApiPlatform\Metadata\Get;
+use ApiPlatform\Metadata\GetCollection;
+use ApiPlatform\Metadata\Patch;
+use ApiPlatform\Metadata\Post;
+use ApiPlatform\Metadata\Put;
+use App\Repository\WebsiteMediaRepository;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Serializer\Attribute\Groups;
+
+#[ORM\Entity(repositoryClass: WebsiteMediaRepository::class)]
+#[ApiResource(
+    normalizationContext: ['groups' => [self::READ]],
+    denormalizationContext: ['groups' => [self::WRITE]],
+    extraProperties: [
+        'standard_put',
+    ])]
+#[Get]
+#[Patch(security: "is_granted('ROLE_ADMIN')")]
+#[GetCollection]
+#[Post(security: "is_granted('ROLE_ADMIN')")]
+#[Delete(security: "is_granted('ROLE_ADMIN')")]
+#[ApiFilter(SearchFilter::class, properties: ['id' => 'exact'])]
+final class WebsiteMedia
```

Comment:
```suggestion
class WebsiteMedia
```
It's why the test fails